receiver/mysqlreceiver/documentation.md

```diff
@@ -26,6 +26,8 @@ These are the metrics available for this scraper.
 | **mysql.row_locks** | The number of InnoDB row locks. | 1 | Sum(Int) | <ul> <li>row_locks</li> </ul> |
 | **mysql.row_operations** | The number of InnoDB row operations. | 1 | Sum(Int) | <ul> <li>row_operations</li> </ul> |
 | **mysql.sorts** | The number of MySQL sorts. | 1 | Sum(Int) | <ul> <li>sorts</li> </ul> |
+| **mysql.statement_event.count** | Summary of current and recent statement events. | 1 | Sum(Int) | <ul> <li>schema</li> <li>digest</li> <li>digest_text</li> <li>event_states</li> </ul> |
+| **mysql.statement_event.wait.time** | The total wait time of the summarized timed events. | ns | Sum(Int) | <ul> <li>schema</li> <li>digest</li> <li>digest_text</li> </ul> |
```

Comment:
Are these metrics important enough to be enabled y default? Looks like they have pretty big carnality which may result in an unexpected load increase for current users. If they are not critical for mysql monitoring, it maybe makes sense to make them optional.

Reply:
Good point. I agree these should be disabled by default.

Reply:
I will disable them by defautl